pkg/apis/events/v1/cloudbuildsource_defaults.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2020 Google LLC.
```

Comment:
agreed offline to remove all CloudBuildSource files

Reply:
removed.